source/index.rst

```diff
@@ -14,15 +14,15 @@ Open5e aims to be the best open-source resource for 5e content.
    Some content you may be expecting was not released as part of the `5e SRD <http://media.wizards.com/2016/downloads/SRD-OGL_V1.1.pdf>`_,
    so we aren't allowed to include it in this site.
 
-   We are hoping to fill in the gaps using quality homebrew and 3rd party OGL content. 
+   We are hoping to fill in the gaps using quality homebrew and 3rd party OGL content.
    `Contact us <mailto:gm@5esrd.com>`_ or `make a pull request <https://github.com/eepMoody/open5e>`_ if you know something we should add!
-   
+
    .. rst-class:: source
 
 What are we about?
 ------------------
 
-In addition to an open, readable, and easily searchable resource for the 5e SRwe have lots of new things we'd like to add! At the top of our list are:
+In addition to an open, readable, and easily searchable resource for the 5e SRD we have lots of new things we'd like to add! At the top of our list are:
```

Comment:
While we're here, I feel like there should be a comma in between "SRD" and "we".